### PR TITLE
Fix exercises

### LINKED ISSUE #511: Exercise sheet 2, task 3 b) is too difficult
image

3 b) is too difficult especially because 

> Hint: Apply the same steps as we used in the lecture for the logistic regression

Is not correct. There was nothing of the sort in the videos. (I assume it used to be)

### PR DESCRIPTION
**Pull request by tpielok**
closes #511 and #509

### CODE REVIEW

File: exercises/day2/exercises/ex_softmax.Rnw [hunk 1]
```diff
@@ -1,8 +1,7 @@
 \begin{enumerate}
   \item[a)] What is the relationship between softmax $\pi_k(x)=\frac{\exp(\theta_k^T x)}{\sum^g_{j=1} \exp(\theta_j^T x)}$ and the logistic function $\pix =\frac{1}{1+\exp(\theta^{T} x)}$ for $g = 2$ (binary classification)?
 
-  \item[b)] Derive the loss function of the softmax regression. Suppose there are $n$ instances and $g$ features. (Hint: Apply the same steps as we used in the lecture for the logistic regression)
-How does this loss function relate to the negative log likelihood?
+  \item[b)] Derive the likelihood of $n$ multinomially distributed ($g$ features) target variables when the posterior class probabilities are modeled with softmax regression. How can you transform this likelihood function to an empirical risk function? State the associated loss function.
```
Comment (HeidiSeibold) [inline exercises/day2/exercises/ex_softmax.Rnw:4]: I would give a hint here on the steps. Maybe also a formula they can start with?

**Comment by HeidiSeibold** [comment]
Thanks, looks good!

**Comment by HeidiSeibold** [comment]
Now we only need to update the PDFs so that the new versions are also shown on the website :)

**Comment by tpielok** [comment]
I'm on it! :)

**Pull request merged**